theme/layouts/_breadcrumbs.ejs

```diff
@@ -0,0 +1,23 @@
+<%
+const query = site.queryPages;
+if (page.pathname !== '') {
+  const [queryPage] = query(({pathname}) => page.pathname === pathname);
+  const pathnameParts = queryPage.originalPathname.split('/'); %>
+  
+  <nav aria-label='Breadcrumb' id='breadcrumb'>
+  <ol>
+    <% 
+    for (let i=0; i <= pathnameParts.length; i++) {
+      const pickOriginalPathname = pathnameParts.slice(0, i).join("/");
+      const [{pathname, title}] = query(({originalPathname}) => originalPathname == pickOriginalPathname);
+      if (i == pathnameParts.length) { %>
```

Comment:
Suggestion: We already have this path part before defining the loop, maybe we can move it outside the loop, to make the code? I think that way we can make code also more readable. What you think?
```
for (let i=0; i < pathnameParts.length; i++) {
  const parentOriginalPathname = pathnameParts.slice(0, i).join("/");
}
<li aria-current="page'"><%= page.title %></li>
```